person.rb

```diff
@@ -29,6 +30,10 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] As I mentioned for the `add_rental` method in the `Book` class, the same thing happens with the `Person` class, and you may want to create a new Rental for the person so your code may look something like:
```
def add_rental(book, date)
  Rental.new(date, book, self)
end
``` 
So that way, you are passing the person to the rental class as a parameter and you are creating a new one with that information so you have the "has-many" association set again, because one person can have more than one rentals.